junit-common/src/main/java/org/jboss/weld/junit/MockBean.java

```diff
@@ -173,6 +180,10 @@ public boolean isAlternative() {
         return alternative;
     }
 
+    boolean isSelectForSyntheticBeanArchive() {
```

Comment:
`isSelect` -> `isSelected`?

Reply:
Well, at the time you call this method the bean is not selected yet. It's a hint for `WeldInitiator` to select the bean if `isAlternative() == true`. Also it's package-private so users should never use it.

Reply:
Didn't mean it for user, more like when we see that code after some time. But that's really just minor.

Reply:
I will add some javadoc.